CONTRIBUTING.md

```diff
@@ -63,6 +63,27 @@ Certain resources, such as CSS and JavaScript files, need to be served from a st
 python manage.py collectstatic
 ```
 
+### Populate initial data
+
+During the first setup, it's useful to import hardcoded initial entries. In this case,
+there are two [fixtures](https://docs.djangoproject.com/en/3.2/howto/initial-data/):
+
+* Sample threads, located in `project/data/sample_threads.json`
+* Sample categories, located in `project/data/categories.json`
+
+Run the following commands to load fixtures:
+
+```py
+python manage.py loaddata ./data/sample_threads.json
+python manage.py loaddata ./data/categories.json
```

Comment:
To show an example from the `sample_threads.json` [file](https://github.com/CiviWiki/OpenCiviWiki/blob/develop/project/data/sample_threads.json), which will be updated soon:
```json
    {
        "model": "api.thread",
        "fields": {
            "title": "Liquid Fluoride Thorium Reactors",
            "summary": "...",
            "category_id": "2",
            "author_id": "2",
            "created": "2016-10-09 21:07:16.848433+00"
        }
    },
```
Notice that in order to load thread data, we use `category_id` field, so it is expected to be loaded `category` data already. Could you write the line to load `category` data just above the line for `thread` data?